packages/checkbox/src/vaadin-checkbox.js

```diff
@@ -184,6 +184,21 @@ class Checkbox extends SlotLabelMixin(
     return this.$.noop;
   }
 
+  /**
+   * Override __copyNodesToSlotTarget from SlotTargetMixin to show a warning.
+   * @override
+   * @protected
+   * @param {Array<Node>} nodes
```

Comment:
If every node is expected not to be `null`, then the exclamation mark is probably required.

```suggestion
   * @param {!Array<!Node>} nodes
```